### ISSUE TITLE
Loss curves

### ISSUE THREAD

**Issue by Tomsen1410** [YuanGongND/ssast#5]
Hey, great work!
I just wanted to ask whether you might have the loss curves of your runs so that I can compare with my experiments a little bit?

**Comment by YuanGongND** [comment]
Hi Tom,

Thanks for your interest.

I didn't plot the loss curve, but I think I still have (pre)-training logs on my server. Which experiment are you looking for? 

-Yuan

**Comment by Tomsen1410** [comment]
Thanks for the reply! I am looking for the pre-training run, i.e. the one trained on Librispeech + AudioSet.

**Comment by Tomsen1410** [comment]
Bump!
Sorry to get back to the question. But I am currently trying to implement SSAST (well, its masked autoencoder counterpart, see https://arxiv.org/pdf/2203.16691.pdf) for music. For efficiency reasons, I tried to have the computed spectograms in FP16 format, but the reconstruction (generative) loss curve seems a bit weird. It first goes down really quickly, rises again to a given point and drops slowly again afterwards.

I just wanted to have some comparison in order to know what I should expect. Thanks in advance!

**Comment by YuanGongND** [comment]
Hi there, 

I think this is our [log](https://drive.google.com/file/d/1tI3dRYu-YwPMRvJ4aG-ZRVJYOUeeGauL/view?usp=sharing). Unfortunately, I don't think we logged the generation loss but just the discrimination loss. The columns are defined at 
https://github.com/YuanGongND/ssast/blob/35ae7abbdd2870c008feed4ece8b7c6457421b17/src/traintest_mask.py#L147

For your question

>  reconstruction (generative) loss curve seems a bit weird. It first goes down really quickly, rises again to a given point and drops slowly again afterwards.

Could it be possible due to you add L_g and L_d together? Otherwise, the L_g on the training set should always drop.

-Yuan
